types.d.ts

```diff
@@ -168,8 +168,9 @@ declare module 'lmnt-node' {
     /**
      * @param apiKey The API key to use for the connection. Obtain a key
      *     from the [LMNT account page](https://app.lmnt.com/account).
+     * @param baseUrl The base URL for the LMNT API. Defaults to `https://api.lmnt.com`.
```

Comment:
Rather than stating the default value (prone to change, not something we want a client to have to worry about generally) suggest just noting "Defaults to the standard production LMNT base url."